admin-frontend/open_admin_app/lib/widgets/features/table-expanded-view/json/edit_json_value_container.dart

```diff
@@ -36,7 +38,7 @@ class _EditJsonValueContainerState extends State<EditJsonValueContainer> {
     final valueSource = widget.rolloutStrategy != null
         ? widget.rolloutStrategy!.value
         : widget.strBloc.featureValue.valueJson;
-    tec.text = (valueSource ?? '').toString();
+    tec.text = (const JsonEncoder.withIndent('  ').convert(json.decode(valueSource)) ?? '').toString();
```

Comment:
i would put a try/catch in here just in case? then you can set it to empty and editing the page won't blow up.